include/boost/dll/export.hpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * export.hpp
+ *
+ *  Created on: 05.10.2015
+ *      Author: Klemens
+ */
+
+#ifndef INCLUDE_BOOST_DLL_EXPORT_HPP_
+#define INCLUDE_BOOST_DLL_EXPORT_HPP_
+
+#include <boost/config.hpp>
+
+
+#if defined(BOOST_GCC)
```

Comment:
It's better to use Boost.Predef and test for Clang compiler on Linux platform too.